SU-HKKU/cardinal_service_profile
# CSD-427 Updated styled and config based on QA from Sep 4-9

# READY FOR REVIEW

# Summary
-  Updated styled and config based on QA from Sep 4-9

# Needed By (Date)
- By before's working session would be nice :) 

# Urgency
- medium

# Steps to Test
1. Pull the PR
2. Clear cache
3. Check for the following:
- Course code should be in uppercase, modular-spacing(-1), and bolded
- Not Eligible should be digital red, inherit caption styles
- Spotlight cards and view image should be 200x200
- Added sponsor block and styles
- Move Author name for spotlight view underneath the quote.
- Mailto icon is now inline
- Quote font-weight should be 600

# Associated Issues and/or People
- [CSD-427](https://stanfordits.atlassian.net/browse/CSD-427)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
